package.json

```diff
@@ -34,5 +63,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "devDependencies": {
+    "@capacitor/cli": "3.1.1"
+  },
+  "description": "An Ionic project"
```

Comment:
Need to change the description to appropriate one. Ex: A Portfolio application skeleton to get quickly start with portfolio managment application.